bolt-connection/src/channel/browser/browser-channel.js

```diff
@@ -171,6 +171,16 @@ export default class WebSocketChannel {
     })
   }
 
+  /**
+   * Setup the receive timeout for the channel.
+   *
+   * Not supported for the browser channel.
+   *
+   * @param {number} receiveTimeout The amount of time the channel will keep without receive any data before timeout (ms)
+   * @returns {void}
+   */
+  setupReceiveTimeout (receiveTimeout) {}
+
```

Comment:
Most certainly out of scope of this PR, but it should be possible to implement a timeout mechanism for websockets with a wrapper that uses `window.setTimeout`. I wouldn't be surprised if someone has published a library or code-snippets for exactly this.

Reply:
It could be done something like keep a var the timestamp with the last received message and check every `receiveTimeout` if the (current timestamp - last received message timestamp) > receiveTimeout.

Reply:
Yes. Or have only one timeout that gets canceled and reset every time something is sent over the socket. And when it ever triggers the calback that's equivalent of a socket timeout.

Reply:
But as said, seems very much out of scope of this PR.